src/garage/np/algos/base.py

```diff
@@ -25,4 +28,34 @@ def train(self, runner):
             The average return in last epoch cycle or None.
 
         """
-        pass
+
+    def evaluate_performance(self, batch):
+        # pylint: disable=no-self-use
+        r"""Evaluate the performance of the algorithm.
+
+        Args:
+            batch (dict): Evaluation trajectories, representing
+                the best current performance of the algorithm, with key
+                * itr (int): Iteration number.
+                * num_trajs (int): Number of trajectories.
+                * episode_reward_mean (float): Average episode reward.
+                * average_discounted_return (float): Average discounted return.
+                * undiscounted_returns (numpy.ndarray): A numpy array of shape
+                    :math:`(N \bullet [T])` containing the undiscounted returns
+                    for all time steps in this batch.
+
+        """
+        tabular.record('Evaluation/Iteration', batch['itr'])
```

Comment:
you will need to update several parts of the codebase which assume this is named `Itr` -- i think it's Runner or Experiment

Reply:
you can ask @naeioi

Reply:
@naeioi Did you get the point here?

Reply:
`Iteration` is not used in anywhere anymore, but other metrics are used in benchmarks. See for example this file https://github.com/rlworkgroup/garage/blob/c55683cd383ec00f008e5871fece772e96ee93e2/tests/benchmarks/garage/tf/algos/test_benchmark_ddpg.py#L117-L118

Benchmarks should be the only place these metrics are accessed by their logged named.

Reply:
Isn't this just label name string? What changes do we need to do?

Reply:
It has to match the name when logged using `tabular.record(name, value)`.

Reply:
I was wrong about `Iteration` not being used anywhere. Actually, there are many references in benchmark files. e.g. in https://github.com/rlworkgroup/garage/blob/c55683cd383ec00f008e5871fece772e96ee93e2/tests/benchmarks/garage/tf/policies/test_benchmark_categorical_cnn_policy.py#L72

They need to be renamed to `Evaluation/Iteration` if the iteration is logged under this name.

Reply:
@naeioi do you use `Iteration` to log alternative x-axis?

Reply:
I don't. These are old benchmarks. I use 'TotalEnvSteps' now.